office365_admin/help.md

```diff
@@ -1,18 +1,15 @@
 # Description
 
-[Office 365](https://www.office.com/) is a SaaS version of Microsoft’s popular Microsoft Office productivity suite. This plugin utilizes the [Microsoft Graph API](https://developer.microsoft.com/en-us/graph/docs/concepts/overview) to manage Office 365 users and licenses.
+Administrate and manage Office 365 users and licenses
 
 # Key Features
 
-* Add users, assign licenses, and update usage locations to automate the provisioning and management of user accounts in your Office 365 subscription
-* Delete users from your Office 365 subscription
-* Get a list of subscription SKUs to maintain licensing records
+Identify key features of plugin.
```

Comment:
It looks like Key features got overwritten. possibly a bug with the regen

Reply:
yes a regen bug.  It messes up the examples as well.  Ty for the catch